packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -16,8 +16,6 @@ import { handleBackupLog } from '../../_handleBackupLog'
 import { unboxIdsFromPattern } from '../../utils'
 import { waitAll } from '../../_waitAll'
 
-import { translateLegacyJob } from './migration'
```

Comment:
`migration.js` can be removed.

Reply:
I thought I did it, I must have messed up during a rebase.

Thanks :+1: